qiita_pet/static/js/sharing.js

```diff
@@ -1,5 +1,3 @@
-var current_study = null;
-
 $(document).ready(function () {
```

Comment:
Actually I just saw this - this is a library, why does it has a `$(document).ready` call? It seems weird that this is in the library and this may be the source of the difficulties that you where finding. If this is used to initialize the different parts of this library, it may be useful to actually have a `sharing_init` call that you call in the `$(document).ready` of the actual HTML pages that is using this library.